dependencies.yaml

```diff
@@ -238,7 +238,7 @@ dependencies:
     common:
       - output_types: [conda, requirements, pyproject]
         packages:
-          - cython>=0.29,<0.30
+          - cython>=3.0.0
```

Comment:
Can you add clang-tools to this dependencies.yaml?

Reply:
Yep, can do

Reply:
@harrism it [looks like](https://github.com/rapidsai/cuspatial/blob/branch-23.10/.pre-commit-config.yaml#L34) the pre-commit hooks use clang-format 15. Should I file a follow-up to re-format everything with clang-format 16?

Reply:
That seems fine, if it's no trouble.